.editorconfig

```diff
@@ -0,0 +1,14 @@
+# editorconfig.org
+
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+insert_final_newline = true
+indent_style = space
+indent_size = 4
+trim_trailing_whitespace = true
+
+[*.md]
+trim_trailing_whitespace = false
```

Comment:
No, you see the following scenario : 
```
hello
world
```
Output: 
-> hello world

```
hello__
world
```
Output: 
-> hello 
world
 (where underscores are pretending to be spaces)

For more details this [here](http://area51.phpbb.com/phpBB/viewtopic.php?t=46077)